tdrs-frontend/Dockerfile

```diff
@@ -1,35 +1,36 @@
-# This Dockerfile is intended for use in deployments that target the
-# `development` environment within Cloud.gov
 
-# ---
 # Stage 0: Create the app directory, install dependencies, and copy in the source code
 FROM node:12.18-alpine3.12 AS localdev
 
+USER node
+
 RUN mkdir /home/node/app/ && chown -R node:node /home/node/app
 WORKDIR /home/node/app
+
+# Copy package.json and run yarn install before copying the rest of the app in
+# to allow Docker to cache the yarn layer to prevent unnecessary installations
+# when code changes occur
 COPY --chown=node:node package.json yarn.lock ./
-USER node
-RUN yarn install && yarn cache clean
 
-# Override the default backend URL to point to the `development` environment
-RUN touch .env.production
-RUN echo "REACT_APP_BACKEND_URL=https://tdp-backend.app.cloud.gov/v1" >> .env.production
-RUN echo "REACT_APP_BACKEND_HOST=https://tdp-backend.app.cloud.gov" >> .env.production
+# Install yarn packages, using local cache if available and skip creating a
+# new lock file to speed up the build
+RUN yarn install --prefer-offline --frozen-lockfile
 
-COPY --chown=node:node . .
 
+COPY --chown=node:node . .
 
+ENV SASS_PATH=node_modules:src
+ENV PORT=80
+EXPOSE 80
 
 # ---
 # Stage 1: Create a production build
 FROM localdev AS build
 RUN yarn build
 
-
-
 # ---
 # Stage 2: Serve over nginx
-FROM nginx:1.19-alpine
+FROM nginx:1.19-alpine AS nginx
```

Comment:
The default docker compose will target this stage, allowing the nginx server to be initialized as it was in the separate dockerfiles.